R/github.R

```diff
@@ -165,8 +165,28 @@ github_error <- function(res) {
           "Use `usethis::edit_r_environ()` and add the token as `GITHUB_PAT`."
         }
       )
+  } else if (identical(as.integer(res$status_code), 404L)) {
+    repo_information = re_match(res$url,"(repos)/(?P<owner>[^/]+)/(?P<repo>[^/]++)/")
+    pat_guidance <- sprintf(
```

Comment:
Can you rename `pat_guidence` to just `guidence`. It is now not always about the PAT.

Reply:
Done.